### PR TITLE
feat(chat-e2e): select searched entities tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

select searched entities tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1870.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1870.nightly-test.deltixhub.io
>E2E tests status: success

**Review by denys-kolomiitsev — DISMISSED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1870.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/ui/webElements/folders.ts [hunk 1]
```diff
@@ -383,22 +386,28 @@ export class Folders extends BaseElement {
   public getFolderEntityCheckbox(
     folderName: string,
     entityName: string,
-    index?: number,
+    folderIndex?: number,
+    entityIndex?: number,
```
Comment (nartovm) [inline apps/chat-e2e/src/ui/webElements/folders.ts:393]: If we are counting from 1 and not from zero, we should start adding some check to ensure that the provided entityIndex is valid (at least not zero?)

Comment (irinakartun) [reply apps/chat-e2e/src/ui/webElements/folders.ts:393]: I guess this is responsibility of test author to use a valid index. Do you mean I should add smth like this?
`const index = entityIndex ? entityIndex === 0 ? 0 : entityIndex - 1 : 0;`

Comment (irinakartun) [reply apps/chat-e2e/src/ui/webElements/folders.ts:393]: throw an error in case of zero index

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1870.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by nartovm — APPROVED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1870.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1870.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by denys-kolomiitsev — APPROVED** [review]

**Pull request merged**